### PR TITLE
Upgrade to latest Clerk and Tooling

### PR DESCRIPTION
**Pull request by JacobMGEvans**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+  SignInButton,
+  SignedIn,
+  SignedOut,
+  UserButton,
+} from '@clerk/clerk-react';
 
 function App() {
-
   return (
     <div>
```
Comment (royanger) [inline src/App.tsx]:
```suggestion
    <header>
```
Change to match the Quickstart guide

File: src/App.tsx [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
       <SignedOut>
         <SignInButton />
       </SignedOut>
       <SignedIn>
-        <UserButton afterSignOutUrl="/" />
+        <UserButton />
       </SignedIn>
     </div>
```
Comment (royanger) [inline src/App.tsx]:
```suggestion
    </header>
```
Change to match the Quickstart guide

File: src/App.tsx [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
 function App() {
-
   return (
     <div>
       <SignedOut>
         <SignInButton />
       </SignedOut>
       <SignedIn>
-        <UserButton afterSignOutUrl="/" />
+        <UserButton />
```
Comment (royanger) [inline src/App.tsx:15]: I think we should keep the `afterSignOutUrl` prop and PR the Quickstart guide to match.

File: package.json [hunk 4]
```diff
@@ -6,24 +6,29 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint . --report-unused-disable-directives --max-warnings 0",
```
Comment (royanger) [inline package.json:9]: If we're adding eslint, we should probably add prettier too at the same time.

**Review by royanger — CHANGES_REQUESTED** [review]
Added a couple of changes (the 'header' suggestions) and a couple of thoughts/questions.

File: src/App.tsx (hunk 3 above)
Comment (JacobMGEvans) [reply src/App.tsx:15]: I rather change the quickstart, this prop is deprecated. I can do that first thing Monday

File: package.json (hunk 4 above)
Comment (JacobMGEvans) [reply package.json:9]: I didnt add anything, I just updated it. I am more than happy to include Prettier too though!

File: src/App.tsx (hunk 3 above)
Comment (royanger) [reply src/App.tsx:15]: I forgot this prop was moved to `<ClerkProvider />`. Let's move it there in both this repo and the guide. There is deffinitely value in including it.

File: package.json (hunk 4 above)
Comment (royanger) [reply package.json:9]: Sorry, I saw the new file and missed that you were just moving it.

File: src/App.tsx (hunk 3 above)
Comment (JacobMGEvans) [reply src/App.tsx:15]: For sure!

File: package.json (hunk 4 above)
Comment (JacobMGEvans) [reply package.json:9]: Still happy to add Prettier if you like 😆

Comment (royanger) [reply package.json:9]: Yes, let's try and add prettier to all of these repos. It will keep diffs cleaner.

File: src/App.tsx (hunk 3 above)
Comment (JacobMGEvans) [reply src/App.tsx:15]: Done and related Docs PR https://github.com/clerk/clerk-docs/pull/1269

**Review by royanger — APPROVED** [review]
🚀

**Pull request merged**